pages/nfttest/mint/_iscnId.vue

```diff
@@ -272,6 +274,79 @@ export default class NFTTestMintPage extends Vue {
     }
   }
 
+  async getOgImage() {
+    try {
+      const { url } = this.iscnData.contentMetadata
+      if (!url) { return }
+      const { data: { image } } = await this.$axios.get(`/crawler/?url=${encodeURIComponent(url)}`)
```

Comment:
Can we pass this information from last page if they came from the fetch page?
We can fetch the API again if this info does not exists (ie did not came from fetch page)